server/channels/db/migrations/postgres/000110_create_desktop_tokens.up.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS desktoptokens (
```

Comment:
What will happen to this migration? Once deployed, this table has already been created. We would need to sync up with the SRE team to actually drop the table, and let it be re-created again.

Reply:
I guess the easiest thing to do would be to just keep the original migration in place, then do an ALTER to make the changes. I'll do that.